BackEnd/tests/unit/test_s3.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import app.util.aws.s3 as s3
+import boto3
+import io
+
+FILE_NAME = 'test_user/housing/testFile.txt'
+OBJ_NAME = 'test_user/housing/testFile.txt'
+BUCKET = 'houseit'
+USER_NAME = 'test_user'
+PREFIX = 'test_user/housing'
+IMG1 = 'app/assets/room_mock_images/1.jpg'
+IMG2 = 'app/assets/room_mock_images/2.jpg'
+IMG3 = 'app/assets/room_mock_images/3.jpg'
+IMG4 = 'app/assets/room_mock_images/4.jpg'
+
+class TestS3(unittest.TestCase):
+
+    def test_upload_and_delete_file_wname(self):
+        self.assertEqual(s3.upload_file_wname(FILE_NAME, BUCKET), True)
+        self.assertEqual(s3.delete_file_wname(FILE_NAME, BUCKET), True)
+
+        self.assertEqual(s3.upload_file_wname(FILE_NAME, BUCKET, OBJ_NAME), True)
+        self.assertEqual(s3.delete_file_wname(FILE_NAME, BUCKET), True)
+
+    def test_upload_file_wobject(self):
+        with open(FILE_NAME, 'rb') as FILE_OBJ:
+            self.assertEqual(s3.upload_file_wobject(FILE_OBJ, BUCKET, OBJ_NAME), True)
+            
+        with open(FILE_NAME, 'rb') as FILE_OBJ:  
+            self.assertEqual(s3.upload_file_wobject(FILE_OBJ, BUCKET), True)
+
+
+    def test_delete_folder(self):
+        # when folder does not exist
+        self.assertEqual(s3.delete_folder(PREFIX, BUCKET), True)
+
+        # when folder exists
+        s3.upload_file_wname(FILE_NAME, BUCKET, OBJ_NAME)
+        self.assertEqual(s3.delete_folder(PREFIX, BUCKET), True)
+
+    def test_get_images(self):
+        unused_category = 'abcd'
```

Comment:
nitpicking: to keep style consistent, we can have this as a ALL CAP constant on the very top